File: projects/average.c
//Design a program that receives a list of numbers using argvargument and printthe average value in the console.//


#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    int x = 0;
    for (int i = 1; i < argc; i++){
        x = x + atof(argv[i]);
    }

    float listNum = argc-1;
    float average = x/listNum;
    printf("the average is: %f\n",average);
}

File: projects/Average.py
#Design a program that receives a list of numbers using argv argument and print the
#average value in the console. 

from sys import argv 

if __name__ == "__main__":
    listNum=[]
    for i in range(1, len(argv)):
        listNum.append(int(argv[i]))

    average = (sum(listNum))/(len(listNum))
    print("The average of the numbers is:", average) 